packages/modules/error-handling-react/package.json

```diff
@@ -38,7 +38,7 @@
     "build/**/*"
   ],
   "dependencies": {
-    "@itwin/itwinui-react": "^1.16.2",
+    "@itwin/itwinui-react": "^2.6.0",
```

Comment:
can you revert changes to this pkg?
this pkg.json and the FallbackUi.tsx

I was under the impression this pkg didnt get compiled at all, when run in the viewer repo
It might in a cospace, but need to update the pnpm-workspace.yaml

Reply:
reverted